plugins/android/src/main/java/androidProjectConfiguration.kt

```diff
@@ -42,7 +44,54 @@ fun Project.androidProjectConfiguration(
         mandatoryOwners = mandatoryOwners
     )
 
-    Forma.store(configuration)
+    Forma.defaultConfiguration = configuration
+
+}
+
+/**
+ * Add configuration with specific [FormaConfigurationKey].
+ * If configuration argument is null value set in [androidProjectDefaultConfiguration] will be taken.
+ * Also [repositories] will be added to repositories setted in [androidProjectDefaultConfiguration]
+ */
+fun androidProjectConfiguration(
+    configurationKey: FormaConfigurationKey,
+    minSdk: Int? = null,
+    targetSdk: Int? = null,
```

Comment:
Passing null as default value means, that parameters become optional. And I don't sure, that all parameters here are as such)

Reply:
In this case I guess optional parameters is ok since we only overriding values from default config